app/assets/javascripts/3dbio_viewer/src/webapp/components/BasicInfoEntry.tsx

```diff
@@ -0,0 +1,137 @@
+import _ from "lodash";
+import React from "react";
+import { Pdb, PdbPublication } from "../../domain/entities/Pdb";
+import { DbItem, Selection, buildDbItem } from "../view-models/Selection";
+import i18n from "../utils/i18n";
+
+export interface BasicInfoProps {
+    pdb: Pdb;
+    selection: Selection;
+    setSelection: (newSelection: Selection) => void;
+}
+
+interface Item {
+    name: string;
+    value?: string;
+    links?: {
+        value: string;
+        href?: string;
+        itemToAdd?: DbItem;
+    }[];
+}
+
+export const BasicInfoEntry: React.FC<BasicInfoProps> = React.memo(props => {
+    const { pdb, selection, setSelection } = props;
+
+    const addOverlayItem = React.useCallback(
+        (item: DbItem) => () =>
+            selection.type === "free" &&
+            setSelection({
+                ...selection,
+                overlay: [...selection.overlay, item],
+            }),
+        [setSelection, selection]
+    );
+
+    const items: Item[] = React.useMemo(
+        () =>
+            (pdb.publications[0] &&
+                getItems(pdb.publications[0]).filter(item => item.value || item.links)) ??
+            [],
+        [pdb.publications]
+    );
+
+    return (
+        <ul>
+            {items.map(({ name, value, links }, idx) => (
+                <li key={idx}>
+                    {!links && (
+                        <span>
+                            {name}: {value ?? "-"}
+                        </span>
+                    )}
+                    {links && (
+                        <span>
+                            {name}:{" "}
+                            {links.map(({ value, href, itemToAdd }, idx) => {
+                                if (href)
+                                    return (
+                                        <React.Fragment key={idx}>
+                                            <a
+                                                key={idx}
+                                                href={href}
+                                                target="_blank"
+                                                rel="noreferrer noopener"
```

Comment:
We have this <a + rel + target many times, we can abstract to generic component.